### ISSUE TITLE
Cannot create batch update account transaction

### ISSUE THREAD

**Issue by SimiHunjan** [hashgraph/hedera-transaction-tool-demo#527]
### Description

1. Created an account update with three accounts
2. Exported the transaction to the input files folder
3. The account update batch transaction did not show up on the home screen
4. Closed the application
5. Tried to open the application and the application will not open now
6. Verified the files were in the input files folder

### Steps to reproduce

See above 

### Hedera network

other

### Version

0.14.0-alpha.5

### Operating system

None

**Comment by SimiHunjan** [comment]
```
2023-10-30 18:39:30.461 [66910] [JavaFX Application Thread] DEBUG com.hedera.hashgraph.client.core.remote.LargeBinaryFile.<init>(LargeBinaryFile.java:169) - File unzipped
2023-10-30 18:39:30.463 [66910] [JavaFX Application Thread] ERROR com.hedera.hashgraph.client.core.remote.LargeBinaryFile.handleError(LargeBinaryFile.java:407) - Missing transaction valid start
2023-10-30 18:39:30.471 [66910] [JavaFX Application Thread] INFO  com.hedera.hashgraph.client.core.remote.RemoteFilesMap.getRemoteFileList(RemoteFilesMap.java:211) - File will be ignored: /Users/user/Desktop/2.4.1test/InputFiles/green.pem
2023-10-30 18:39:30.476 [66910] [JavaFX Application Thread] INFO  com.hedera.hashgraph.client.core.remote.RemoteFilesMap.getRemoteFileList(RemoteFilesMap.java:211) - File will be ignored: /Users/user/Desktop/2.4.1test/InputFiles/orange.pem
2023-10-30 18:39:30.477 [66910] [JavaFX Application Thread] INFO  com.hedera.hashgraph.client.core.remote.RemoteFilesMap.getRemoteFileList(RemoteFilesMap.java:211) - File will be ignored: /Users/user/Desktop/2.4.1test/InputFiles/test.pem
2023-10-30 18:39:30.498 [66910] [JavaFX Application Thread] INFO  com.hedera.hashgraph.client.core.remote.PublicKeyFile.duplicate(PublicKeyFile.java:90) - This key has a PEM in the Keys folder. Marked as duplicate
2023-10-30 18:39:30.804 [66910] [JavaFX Application Thread] DEBUG com.hedera.hashgraph.client.ui.Controller.getPublicKeys(Controller.java:1099) - pubFiles loaded
2023-10-30 18:39:30.808 [66910] [JavaFX Application Thread] INFO  com.hedera.hashgraph.client.ui.HomePaneController$1.expireAfterCreate(HomePaneController.java:165) - testCSV1.csv will expire in: 59662349 seconds
2023-10-30 18:39:30.816 [66910] [JavaFX Application Thread] DEBUG com.hedera.hashgraph.client.ui.utilities.KeyStructureUtility.loadPubKeys(KeyStructureUtility.java:89) - Public keys loaded
2023-10-30 18:39:30.878 [66910] [main] ERROR com.hedera.hashgraph.client.core.exceptions.Log4j2UncaughtExceptionHandler.uncaughtException(Log4j2UncaughtExceptionHandler.java:40) - ERROR: uncaught exception occurs, check app.debug.log log file for details. 
Message: Exception in Application start method
2023-10-30 18:39:30.879 [66910] [main] ERROR com.hedera.hashgraph.client.core.exceptions.Log4j2UncaughtExceptionHandler.uncaughtException(Log4j2UncaughtExceptionHandler.java:41) - Uncaught exception on thread Thread[main,5,main]
java.lang.RuntimeException: Exception in Application start method
	at com.sun.javafx.application.LauncherImpl.launchApplication1(Unknown Source) ~[javafx.graphics:?]
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$2(Unknown Source) ~[javafx.graphics:?]
	at java.lang.Thread.run(Unknown Source) [?:?]
Caused by: javafx.fxml.LoadException: 
file:/Applications/TransactionTools.app/Contents/app/transactiontools.jar!/com/hedera/hashgraph/client/ui/UserInterface.fxml

	at javafx.fxml.FXMLLoader.constructLoadException(Unknown Source) ~[javafx.fxml:?]
	at javafx.fxml.FXMLLoader.loadImpl(Unknown Source) ~[javafx.fxml:?]
	at javafx.fxml.FXMLLoader.loadImpl(Unknown Source) ~[javafx.fxml:?]
	at javafx.fxml.FXMLLoader.loadImpl(Unknown Source) ~[javafx.fxml:?]
	at javafx.fxml.FXMLLoader.loadImpl(Unknown Source) ~[javafx.fxml:?]
	at javafx.fxml.FXMLLoader.loadImpl(Unknown Source) ~[javafx.fxml:?]
	at javafx.fxml.FXMLLoader.loadImpl(Unknown Source) ~[javafx.fxml:?]
	at javafx.fxml.FXMLLoader.loadImpl(Unknown Source) ~[javafx.fxml:?]
	at javafx.fxml.FXMLLoader.load(Unknown Source) ~[javafx.fxml:?]
	at com.hedera.hashgraph.client.ui.StartUI.start(StartUI.java:37) ~[transactiontools.jar:0.14.0]
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(Unknown Source) ~[javafx.graphics:?]
	at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(Unknown Source) ~[javafx.graphics:?]
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(Unknown Source) ~[javafx.graphics:?]
	at java.security.AccessController.doPrivileged(Unknown Source) ~[?:?]
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(Unknown Source) ~[javafx.graphics:?]
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(Unknown Source) ~[javafx.graphics:?]
Caused by: java.lang.NullPointerException: Cannot invoke "com.hedera.hashgraph.client.core.remote.helpers.AccountList.getList()" because the return value of "com.hedera.hashgraph.client.core.remote.TransactionCreationMetadataFile.getNodes()" is null
	at com.hedera.hashgraph.client.core.remote.TransactionFile.handleTransactionCommonFields(TransactionFile.java:259) ~[tools-core-0.14.0.jar:?]
	at com.hedera.hashgraph.client.core.remote.TransactionFile.buildGridPane(TransactionFile.java:206) ~[tools-core-0.14.0.jar:?]
	at com.hedera.hashgraph.client.core.remote.RemoteFile.buildDetailsBox(RemoteFile.java:601) ~[tools-core-0.14.0.jar:?]
	at com.hedera.hashgraph.client.ui.HomePaneController.buildAndAddBox(HomePaneController.java:412) ~[transactiontools.jar:0.14.0]
	at com.hedera.hashgraph.client.ui.HomePaneController.loadInputFolderIntoMap(HomePaneController.java:283) ~[transactiontools.jar:0.14.0]
	at com.hedera.hashgraph.client.ui.HomePaneController.loadRemoteFilesMap(HomePaneController.java:253) ~[transactiontools.jar:0.14.0]
	at com.hedera.hashgraph.client.ui.HomePaneController.populatePane(HomePaneController.java:221) ~[transactiontools.jar:0.14.0]
	at com.hedera.hashgraph.client.ui.HomePaneController.initializePane(HomePaneController.java:203) ~[transactiontools.jar:0.14.0]
	at com.hedera.hashgraph.client.ui.Controller.startupPhaseInitialization(Controller.java:407) ~[transactiontools.jar:0.14.0]
	at com.hedera.hashgraph.client.ui.Controller.initialize(Controller.java:306) ~[transactiontools.jar:0.14.0]
	at javafx.fxml.FXMLLoader.loadImpl(Unknown Source) ~[javafx.fxml:?]
	at javafx.fxml.FXMLLoader.loadImpl(Unknown Source) ~[javafx.fxml:?]
	at javafx.fxml.FXMLLoader.loadImpl(Unknown Source) ~[javafx.fxml:?]
	at javafx.fxml.FXMLLoader.loadImpl(Unknown Source) ~[javafx.fxml:?]
	at javafx.fxml.FXMLLoader.loadImpl(Unknown Source) ~[javafx.fxml:?]
	at javafx.fxml.FXMLLoader.loadImpl(Unknown Source) ~[javafx.fxml:?]
	at javafx.fxml.FXMLLoader.loadImpl(Unknown Source) ~[javafx.fxml:?]
	at javafx.fxml.FXMLLoader.load(Unknown Source) ~[javafx.fxml:?]
	at com.hedera.hashgraph.client.ui.StartUI.start(StartUI.java:37) ~[transactiontools.jar:0.14.0]
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(Unknown Source) ~[javafx.graphics:?]
	at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(Unknown Source) ~[javafx.graphics:?]
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(Unknown Source) ~[javafx.graphics:?]
	at java.security.AccessController.doPrivileged(Unknown Source) ~[?:?]
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(Unknown Source) ~[javafx.graphics:?]
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(Unknown Source) ~[javafx.graphics:?]

```

**Comment by SimiHunjan** [comment]
```
2023-10-30 18:39:30.471 INFO  - File will be ignored: /Users/user/Desktop/2.4.1test/InputFiles/green.pem
2023-10-30 18:39:30.476 INFO  - File will be ignored: /Users/user/Desktop/2.4.1test/InputFiles/orange.pem
2023-10-30 18:39:30.477 INFO  - File will be ignored: /Users/user/Desktop/2.4.1test/InputFiles/test.pem
2023-10-30 18:39:30.498 INFO  - This key has a PEM in the Keys folder. Marked as duplicate
2023-10-30 18:39:30.808 INFO  - testCSV1.csv will expire in: 59662349 seconds
2023-10-30 18:39:30.878 ERROR - ERROR: uncaught exception occurs, check app.debug.log log file for details. 
Message: Exception in Application start method
2023-10-30 18:39:30.879 ERROR - Uncaught exception on thread Thread[main,5,main] Exception in Application start method
```

**Comment by SimiHunjan** [comment]
verified fix in 0.14.0-alpha7
